core/src/env/mod.rs

```diff
@@ -40,17 +40,29 @@ mod test_env;
 pub use api::*;
 pub use traits::*;
 
-/// The environment implementation that is currently being used.
+pub use self::srml::DefaultSrmlTypes;
+
+/// The storage environment implementation that is currently being used.
 ///
 /// This may be either
-/// - `DefaultEnv` for real contract storage
+/// - `SrmlEnv` for real contract storage
 ///   manipulation that may happen on-chain.
 /// - `TestEnv` for emulating a contract environment
 ///   that can be inspected by the user and used
 ///   for testing contracts off-chain.
 #[cfg(not(feature = "test-env"))]
-pub(self) type ContractEnv = self::srml::DefaultSrmlEnv;
+pub type ContractEnvStorage = self::srml::SrmlEnvStorage;
```

Comment:
Is this correct being `pub` while the other definitions are `pub(self)` ?

Reply:
You're right it should also be `pub (self)`